cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,23 @@
+class ProductsListPage {
+    private ProductList: string;
+    private AddCart: string
+
+    constructor() {
+        this.ProductList = '.ajax_block_product';
+        this.AddCart = '#center_column a.button.ajax_add_to_cart_button.btn.btn-default'
+
+    }
+
+    public LookProductList(): void {
+       // cy.get(this.ProductList).click()
+         cy.get(this.ProductList).eq(0).click(); //Proposal
+    }
+
+    public goToAddCart(): void {
+       // cy.get(this.AddCart).click()
+       cy.get(this.AddCart).eq(0).click(); //Proposal
```

Comment:
is the call to .eq(0) method necessary?, If so, try to search a unique locator, to not need that method call.